ci-operator/step-registry/cucushift/installer/check/azure/confidential/trustedlaunch/cucushift-installer-check-azure-confidential-trustedlaunch-commands.sh

```diff
@@ -57,69 +57,115 @@ function property_check() {
     [[ "${expected_value}" == "Disabled" ]] && [[ "${actual_value}" == "false" ]] && return 0
     [[ "${expected_value}" == "Enabled" ]] && [[ "${actual_value}" == "true" ]] && return 0
     [[ "${expected_value}" == "${actual_value}" ]] && [[ "${actual_value}" != "null" ]] && return 0
+    [[ "${expected_value}" == "false" ]] && [[ "${actual_value}" == "null" ]] && return 0
```

Comment:
What case need this condition ?

Reply:
In below case, `encryptionAtHost` is set to false
install-config:
```
Platform:
  azure:
    defaultMachinePlatform:
      encryptionAtHost: false
```
The [result](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/56841/rehearse-56841-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-azure-ipi-confidential-trustedlaunch-f28-destructive/1836314735847411712/artifacts/azure-ipi-confidential-trustedlaunch-f28-destructive/cucushift-installer-check-azure-confidential-trustedlaunch/build-log.txt) returned from az command:
```
node ci-op-65bfir4n-b8e7b-v2tcp-master-0 security profile
{
  "encryptionAtHost": null,
  "securityType": "TrustedLaunch",
  "uefiSettings": {
    "secureBootEnabled": true,
    "vTpmEnabled": true
  }
}
```

Reply:
How about this:
```
[[ "${expected_value}" == "false" ]] && ([[ "${actual_value}" == "false" ]] || [[ "${actual_value}" == "null" ]]) && return 0
```

Reply:
> set encryptionAtHost to false under defaultMachinePlatform to make sure that installation is successful in such case.

Why we have to set encryptionAtHost to false under defaultMachinePlatform , some known issue ?

Reply:
yes, please see case 2 in [comments](https://issues.redhat.com/browse/OCPBUGS-36670?focusedId=25559819&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25559819) of bug OCPBUGS-36670, so set encryptionAtHost to false on purpose to test case 2, make sure that gen2 image definition has expected feature setting without encryptionAtHost or encryptionAtHost as false.